src/scripts/main.js

```diff
@@ -1,3 +1,277 @@
 'use strict';
 
-// write your code here
+const cells = document.querySelectorAll('.field-cell');
+const score = document.querySelector('.game-score');
+const startMessage = document.querySelector('.message-start');
+const winMessage = document.querySelector('.message-win');
+const loseMessage = document.querySelector('.message-lose');
+const rows = document.querySelectorAll('.field-row');
+
+function addTile() {
+  const emptyCells = [...cells].filter(cell => !cell.textContent);
+
+  if (emptyCells.length === 0) {
+    return;
+  }
+
+  const randomIndex = Math.floor(Math.random() * emptyCells.length);
+  const randomNumber = Math.random() < 0.9 ? '2' : '4';
+
+  if (randomNumber === '2') {
```

Comment:
Replace the if-else block with a single line: `emptyCells[randomIndex].classList.add('field-cell--' + randomNumber);`